model.py

```diff
@@ -598,6 +598,7 @@ class DataSource(Base):
     ADOBE = "Adobe DRM"
     PLYMPTON = "Plympton"
     OA_CONTENT_SERVER = "Library Simplified Open Access Content Server"
+    GOOGLE = "Google Oauth"
```

Comment:
As I'm thinking about this more I do think it might be better to have it work more like patrons. I think we can say every circulation manager has to have a single admin auth provider, so we don't need to store the data source. We could parse the response from google into optional columns in the admins table, or store the json there.